qemu/tests/blk_commit.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from avocado.core import exceptions
```

Comment:
As avocado-framework/avocado-vt#941 (comment) mentioned, better to use test instance to report the status.

http://avocado-framework.readthedocs.io/en/latest/api/test/avocado.html